### DISCUSSION TITLE
practice-exam/practice-exam-19

### DISCUSSION THREAD

**Discussion by giscus** [kananinirav/AWS-Certified-Cloud-Practitioner-Notes#233] [Comments]
# practice-exam/practice-exam-19

If you are preparing for AWS Cloud Practitioner Exam, this guide will help you with quick revision before the exam. I share the notes I used to study and pass exam.

https://kananinirav.com/practice-exam/practice-exam-19.html

**Comment by hongkai-tan** [comment]
Thank you for providing the notes and these practice questions for free
